pkg/common/utils.go

```diff
@@ -133,3 +136,11 @@ func GetDeviceName(volKey *meta.Key) (string, error) {
 func CreateNodeID(project, zone, name string) string {
 	return fmt.Sprintf(nodeIDFmt, project, zone, name)
 }
+
+func TimeToTimestamp(t time.Time) *timestamp.Timestamp {
```

Comment:
Use the `github.com/golang/protobuf/ptypes` library. It has a method to do this correctly: `func TimestampProto(t time.Time) (*tspb.Timestamp, error)`